site/content/docs/latest/tutorials/kubeapps-on-tce/02-TCE-unmanaged-cluster.md

```diff
@@ -0,0 +1,139 @@
+
+    📦 Selected core package repository
+    projects.registry.vmware.com/tce/repo-12:0.12.0
+
+    📦 Selected additional package repositories
+    projects.registry.vmware.com/tce/main:0.12.0
+
+    📦 Selected kapp-controller image bundle
+    projects.registry.vmware.com/tce/kapp-controller-multi-pkg:v0.30.1
+
+    🚀 Creating cluster kubeapps-cluster
+    Cluster creation using kind!
+    ❤️  Checkout this awesome project at https://kind.sigs.k8s.io
+    Base image downloaded
+    Cluster created
+    To troubleshoot, use:
+
+    kubectl ${COMMAND} --kubeconfig ~/.config/tanzu/tkg/unmanaged/kubeapps-cluster/kube.conf
+
+    📧 Installing kapp-controller
+    kapp-controller status: Running
+
+    📧 Installing package repositories
+    tkg-core-repository package repo status: Reconcile succeeded
+
+    🌐 Installing CNI
+    calico.community.tanzu.vmware.com:3.22.1
+
+    ✅ Cluster created
+
+    🎮 kubectl context set to kubeapps-cluster
+
+    View available packages:
+        tanzu package available list
+    View running pods:
+        kubectl get po -A
+    Delete this cluster:
+        tanzu unmanaged delete kubeapps-cluster
+    ```
+
+3. The new unmanaged cluster is automatically added to your local kubeconfig. If you have [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) installed locally, you can now use it to interact with the cluster.
+
+4. Once the cluster is up and running, let's check that the TCE catalog has been automatically added to it.
+
+    ```bash
+    tanzu package repository list --all-namespaces
+    ```
+
+    And the output should contain the most up-to-date TCE main catalog:
+
+    ```bash
+    NAME                                          REPOSITORY                                TAG     STATUS               DETAILS  NAMESPACE
+    projects.registry.vmware.com-tce-main-0.12.0  projects.registry.vmware.com/tce/main     0.12.0  Reconcile succeeded           tanzu-package-repo-global
+    tkg-core-repository                           projects.registry.vmware.com/tce/repo-12  0.12.0  Reconcile succeeded           tkg-system
+    ```
+
+## Authentication for an unmanaged cluster
+
+Unmanaged clusters are meant for development/testing mainly, and for this tutorial we will use the **service account authentication**.
+
+Please remember that for any user-facing installation you should [configure an OAuth2/OIDC provider](/site/content/docs/latest/tutorials/using-an-OIDC-provider.md) to enable secure user authentication with Kubeapps and the cluster.
+
+### Credentials creation
+
+Let's continue by creating a demo credential with which to access Kubeapps and Kubernetes.
+Service account will be named `kubeapps-operator` and will have `cluster-admin` role.
+
+```bash
+kubectl create --namespace default serviceaccount kubeapps-operator
+kubectl create clusterrolebinding kubeapps-operator --clusterrole=cluster-admin --serviceaccount=default:kubeapps-operator
+cat <<EOF | kubectl apply -f -
```

Comment:
name: kubeapps-operator-token
  namespace: default
  annotations:

Needs indentation, if not copy&paste from the instructions will return an error.
EOF